crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -138,11 +135,21 @@ public static void validateSharedContent(Long boardId, Long sectionId, Long upda
     }
 
     //TODO documentation
-    public static void validateTwoIds(Long firstId, Long secondId){
-        validate(firstId, Regex.ID.getRegex());
-        validate(secondId, Regex.ID.getRegex());
+    public static void validateIDs(List<Long> ids){
```

Comment:
Don't get the arguments as a list.
Use this to figure out the use of ...(spread operator) -> https://stackoverflow.com/questions/46388620/java-spread-operator
Its the correct way